packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -425,6 +430,20 @@ class LoggingController extends DevToolsScreenController
     }
   }
 
+  void _handleTimerEvent(Event e) {
+    final kind = e.kind!;
+
+    log(
+      LogData(
+        kind,
```

Comment:
nit: just use `e.kind!` directly here since the `kind` local var isn't used for anything else